inference/app/models/chat_completion/model.py

```diff
@@ -259,7 +286,10 @@ def prepare_response(
             message = ChatCompletionAssistantMessage(content=text_content or "")
 
         usage = ChatCompletionUsage(
-            input_tokens=input_tokens, output_tokens=estimate_response_tokens(message.model_dump())
+            input_tokens=input_tokens,
+            output_tokens=output_tokens
+            if output_tokens is not None
+            else estimate_response_tokens(message.model_dump()),
```

Comment:
don't use inline statement here

add this before `usage = ChatCompletionUsage`
```
if output_tokens is None:
    output_tokens =  estimate_response_tokens(message.model_dump()),
````